Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/StaticCommonTraitsPlugin.cs

```diff
@@ -39,7 +43,7 @@ private static string ChooseRuntime(IAppArgs appArgs)
                 return UNITY_EDITOR;
 
             if (appArgs.HasFlag(AppArgsFlags.DCL_EDITOR))
-                return AppArgsFlags.DCL_EDITOR;
+                return DCL_EDITOR;
```

Comment:
Could we add comment about it, as we will definitely forget why we are passing dcl-editor instead of hub

Reply:
Also when you  you launch the scene with the command `npm run start -- --explorer-alpha`, the command would return this `decentraland://realm=http://127.0.0.1:8000/&position=-9,-9&local-scene=true&debug=true&hub=false&dclenv=org`
By just checking `appArgs.HasFlag` we will have false positive on when the client was launched from the creator hub or not.

Reply:
yes, great observation, It slipped me

Reply:
Fixed at https://github.com/decentraland/unity-explorer/pull/3583/commits/8779b2a7dc7a57050a4b39584106b98b29f0648b